src/frontend/src/features/layouts/components/thread-panel/components/thread-item/_index.scss

```
@@ -202,14 +202,53 @@
+    white-space: nowrap;
+    pointer-events: none;
     cursor: grabbing;
+    animation: popIn 0.15s var(--c--globals--transitions--ease-in-out);
+    transform-origin: left center;
+}
+
+.thread-drag-preview--exiting {
+    animation: popOut 0.15s var(--c--globals--transitions--ease-in-out) forwards;
+}
+
+@keyframes popIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "popIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)